torch/_tensor_docs.py

```diff
@@ -1899,6 +1899,13 @@ def add_docstr_all(method, docstr):
 See :func:`torch.kthvalue`
 """)
 
+add_docstr_all('ldexp',
+               r"""
+ldexp(other) -> Tensor
+
+See :func:`torch.ldexp`
+""")
+
```

Comment:
Failures are because you need an ldexp_ entry here.

Reply:
Ok I'll fix that! I promise my next operator will be less overhead lol.

Reply:
This is great stuff, don't be discouraged. We've known for awhile that adding an operator to PyTorch is complicated with a lot of bookkeeping. This is one of the simplest ops that can be added, too. Other operations, like the split operations, require modifying several more files and tests. 

We should seriously consider investing in better developer documentation for how to add an operator.